DOCSP-38522: kotlin landing page (#181)
mongodb/docs-landing: source/languages/kotlin.txt

:template: product-landing
:pl-max-width-paragraphs:
:noprevnext:

===================
MongoDB with Kotlin
===================

.. default-domain:: mongodb

.. introduction::
      

>    Develop server-side Kotlin applications with MongoDB's :atlas:`Atlas cloud database </>`. With two Kotlin
>    drivers that support coroutine-based and synchronous development respectively, the MongoDB Kotlin drivers
>    support developers in building server-side applications in Kotlin that are interoperable with the JVM ecosystem.

Reviewer: I see exactly what this means, but I've always seen "respectively" after two lists; e.g., "Apples and oranges are red and orange, respectively." rather than "These two fruits are red and orange, respectively."

```suggestion
   Develop server-side Kotlin applications with MongoDB's :atlas:`Atlas cloud database </>`. With support
   for both coroutine-based and synchronous development, the MongoDB Kotlin drivers
   support developers in building server-side applications in Kotlin that are interoperable with the JVM ecosystem.
```

Author: @caitlindavey I am waiting to change the approved copy until you provide input

Reviewer 2: LGTM
